awx/settings/defaults.py

```diff
@@ -447,16 +429,6 @@ def IS_TESTING(argv=None):
 #     }
 # }
 
-# Use Django-Debug-Toolbar if installed.
-try:
-    import debug_toolbar
-    INSTALLED_APPS += (debug_toolbar.__name__,)
```

Comment:
this isn't necessarily the pattern that works for this anyway. It's just plain broken, I'm planning on fixing it. I just need some time to get around to working on it. I'm fine for it to be removed now.

Reply:
actually I think we may still need this

https://django-debug-toolbar.readthedocs.io/en/latest/installation.html

so it's probably better to not remove it just yet

Reply:
This is all set up in `defaults.py` so I'm going to leave it out.